test.sh

```diff
@@ -530,70 +566,138 @@ function test_make_run_tests() {
+    n_mills=1
+    local n_threads=8
+    msg="Large inserts stress test, ${n_mills}M rows, ${n_threads} threads ${use_msg}"
+    # shellcheck disable=SC2086
+    run_with_timing "${msg}" \
+        "$BINDIR"/unit/large_inserts_bugs_stress_test ${use_shmem} --num-threads ${n_threads}
+
+    # Test runs w/ more inserts and enable bg-threads
+    n_mills=2
+    local n_threads=8
+    msg="Large inserts stress test, ${n_mills}M rows, ${n_threads} threads, 7 bg threads ${use_msg}"
+    # shellcheck disable=SC2086
+    run_with_timing "${msg}" \
+        "$BINDIR"/unit/large_inserts_bugs_stress_test ${use_shmem} \
+                                                      --num-threads ${n_threads} \
+                                                      --num-normal-bg-threads 4 \
+                                                      --num-memtable-bg-threads 3
 }
 
 # ##################################################################
 # Execute a few variations of splinter_test --functionality tests
+# Execute this set w/ and w/o the "--use-shmem" arg.
 # ##################################################################
 function run_splinter_functionality_tests() {
+    local use_shmem=$1
+    local use_msg=
+    if [ "$use_shmem" != "" ]; then
+        use_msg=", using shared memory"
+   fi
+
     key_size=8
-    run_with_timing "Functionality test, key size=${key_size} bytes" \
+    # shellcheck disable=SC2086
+    run_with_timing "Functionality test, key size=${key_size} bytes${use_msg}" \
         "$BINDIR"/driver_test splinter_test --functionality 1000000 100 \
+                                            $use_shmem \
                                             --key-size ${key_size} --seed "$SEED"
 
-    run_with_timing "Functionality test, with default key size" \
+    # shellcheck disable=SC2086
+    run_with_timing "Functionality test, with default key size${use_msg}" \
         "$BINDIR"/driver_test splinter_test --functionality 1000000 100 \
+                                            $use_shmem \
                                             --seed "$SEED"
 
-    run_with_timing "Functionality test, default key size, with background threads" \
+    # shellcheck disable=SC2086
+    run_with_timing "Functionality test, default key size, with background threads${use_msg}" \
         "$BINDIR"/driver_test splinter_test --functionality 1000000 100 \
+                                            $use_shmem \
                                             --num-normal-bg-threads 4 --num-memtable-bg-threads 2 \
                                             --seed "$SEED"
 
     max_key_size=102
-    run_with_timing "Functionality test, key size=maximum (${max_key_size} bytes)" \
+    # shellcheck disable=SC2086
+    run_with_timing "Functionality test, key size=maximum (${max_key_size} bytes)${use_msg}" \
         "$BINDIR"/driver_test splinter_test --functionality 1000000 100 \
+                                            $use_shmem \
```

Comment:
Oh, I seem to have upgraded a few functionality and `--perf` tests, too, (see below) to run with `--use-shmem`.

Even more testing coverage is ON by default in CI!